stance-4forums/README.md

```diff
@@ -0,0 +1,38 @@
+### Stance 4Forums
+
+## Problem
+In this example, we attempt to identify the stance of a speaker in a debate as being for or against.
+
+## Dataset
+
+The dataset comes from two online debate websites: 4forums.com , from the Internet Argument Corpus and createdebate.com . There are 4 topics in the dataset:
```

Comment:
The English is off here.
Commas should not have a space before then, and there should not even be a comma here.

(Same for other stance one.)